app/assets/javascripts/listings/directives/featureRow.js.coffee

```diff
@@ -7,3 +7,4 @@ angular.module('dahlia.directives')
   link: (scope, elem, attrs) ->
     scope.title = attrs.title
     scope.description = attrs.description
+    scope.skiptranslate = attrs.skiptranslate
```

Comment:
whoops I think my comment on this one didn't save. I was wondering where the skiptranslate value for attrs is coming from, is it set somewhere else?

Reply:
nevermind! it's for neighborhood. I see it now.